app/controllers/workflow_executions_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Workflow executions controller
 class WorkflowExecutionsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Is there a reason for why this was added here? This controller inherits from `application_controller.rb` which already has this in it

Reply:
Removed in 5140bacb7d1a4d25da9ef00ae10e58d846292482